sensors/GoalSensor.py

```diff
@@ -0,0 +1,75 @@
+from sensors.SensorCommon import Sensor
+from sensors.SensorCommon import Dist2Circ
+import numpy as np
+import operator
+
+
+class GoalSensor(Sensor):
+
+    def __init__(self, nbGoals=0, SensorRange=10, mode="position"):
```

Comment:
What is the purpose fo this SensorRange variable? I understand it as being the default value when no observation is made (no obstacle for example). Then the naming is confusing.
Could you also be consistent with variables and functions being not-capatilized?

Reply:
Exactly, it is meant to be the default value when no observation is made. Since "zero-padding" might be not the best idea for sensor information I figured it might be best to be able to set it as intended. I will find a more suitable name for it!
Also, I will make naming in general more consistent!

Reply:
Sorry, I mixed up the answer here. SensorRange is not the default value for when no observation is made, but the limit for the observation space. 
For the ObstacleSensor, this aligns with the default value when no obstacle is present. For the GoalSensor, the default value is set to 0. The default value is set in the _reset function.